config/config.go

```diff
@@ -58,90 +40,19 @@ type Config struct {
 	BackendsConfig  json.RawMessage `json:"backends,omitempty"`
 	TasksConfig     json.RawMessage `json:"tasks,omitempty"`
 	TelemetryConfig json.RawMessage `json:"telemetry,omitempty"`
```

Comment:
Yeah, we can remove it - we really don't re-serialize the config anyway, so the extra tag options are mostly useless.